Symfony/CS/Config.php

```diff
@@ -0,0 +1,155 @@
+<?php
```

Comment:
This file was renamed from `Symfony/CS/Config/Config.php`, then:
- change namespace